src/trials/rewardFeedback.js

```diff
@@ -15,7 +15,9 @@ const rewardFeedback = (duration, blockSettings, opts, trialDetails) => {
       let rewards = jsPsych.data.get().select("value").values;
       let last = rewards[rewards.length - 1];
       let stimulus;
-      if (last) {
+        console.log("last:", last);
+        console.log("rewards:", rewards);
+        if (last) {
         stimulus =
```

Comment:
i'm guessing this was the workaround for solving the error when last.rewards or whatever was undefined?